web/api/v1/api.go

```diff
@@ -51,12 +56,13 @@ const (
 type errorType string
 
 const (
-	errorNone     errorType = ""
-	errorTimeout  errorType = "timeout"
-	errorCanceled errorType = "canceled"
-	errorExec     errorType = "execution"
-	errorBadData  errorType = "bad_data"
-	errorInternal errorType = "internal"
+	errorNone        errorType = ""
+	errorTimeout               = "timeout"
```

Comment:
You're undoing a fix I made here, these should all still say errorType. Can you rebase?